environment.yml

```diff
@@ -1,15 +1,17 @@
-name: nf-core-rnafusion-1.0.1
+# You can use this file to create a conda environment for this pipeline:
+#   conda env create -f environment.yml
+name: nf-core-rnafusion-1.0.2
 channels:
-  - bioconda
   - conda-forge
+  - bioconda
   - defaults
 dependencies:
   - anaconda::openjdk=8.0.152 # Needed for FastQC - conda build hangs without this
-  - bioconda::fastqc=0.11.8
   - bioconda::star=2.6.1b # update STAR-Fusion and Fusion-Inspector
-  - bioconda::multiqc=1.7
   - conda-forge::r-data.table=1.12.0
   - conda-forge::r-gplots=3.0.1.1
   - bioconda::bioconductor-edger=3.24.1
   - conda-forge::r-markdown=0.9
   - bioconda::fusion-report=1.0.0
+  - fastqc=0.11.8
```

Comment:
Why did you remove the specification to the bioconda channel for fastQC and multiQC?

Reply:
Again, just following the [template](https://github.com/nf-core/rnafusion/blob/TEMPLATE/environment.yml).

Reply:
That is actually wrong in the template and should be there. Can you open an issue there to fix it so someone can take of it then?

Reply:
Sure, I'll make a PR right away and change this in the pipeline as well!

Reply:
Done

Reply:
Let's keep it here for now until the discussion in the issue is resolved 👍 Don#t want to block it here because of just that, as it works fine and is just a "cosmetic" thing which doesn't break anything.